.merlin

```diff
@@ -5,14 +5,15 @@ S ./src/**
 S ./reUnit/**
 
 ####{BSB GENERATED: NO EDIT
-FLG -ppx /Users/bordoley/personal/immutable-re/node_modules/bs-platform/bin/bsppx.exe
-S /Users/bordoley/personal/immutable-re/node_modules/bs-platform/lib/ocaml
-B /Users/bordoley/personal/immutable-re/node_modules/bs-platform/lib/ocaml
-FLG -nostdlib -no-alias-deps -color always -w -40+6+7+27+32..39+44+45
-S /Users/bordoley/personal/immutable-re/node_modules/reunit/lib/ocaml
-B /Users/bordoley/personal/immutable-re/node_modules/reunit/lib/ocaml
-S perf
-B lib/bs/perf
+FLG -ppx /Users/cem2ran/git/immutable-re/node_modules/bs-platform/lib/bsppx.exe
```

Comment:
It appears that it can be ignored. I've tried removing it and building and testing natively and JS. I'm not sure if this covers all cases w.r.t native workflow I've mostly been using using `bsb-native` where all of .merlin is generated allowing you to ignore the file. Maybe someone with a better knowledge of the native compilation side of things can pitch in on how to solve this.